ipaserver/rpcserver.py

```diff
@@ -72,6 +73,11 @@
 if six.PY3:
     unicode = str
 
+# time.perf_counter_ns appeared in Python 3.7.
+if (version_info.major == 3 and version_info.minor < 7) or \
+   (version_info.major == 2):
```

Comment:
would it be shorter to compare tuples?
```python
if version_info < (3, 7):
```

Reply:
I kinda like the readability of the first version but the 2nd would fit too...

Reply:
Fixed. Thanks.